### DISCUSSION TITLE
How to disable community containers?

### DISCUSSION THREAD

**Discussion by rd-911** [nextcloud/all-in-one#3679] [Questions] [answered]
Hi together, 

I just experimented with some community containers (especially local-ai and pi-hole). As Pi-Hole did not start ("s6-overlay-suexec: fatal: can only run as pid 1", I did not dig deeper into this) and local-ai did not come up with the results I expected, I wanted to revert everything back to my basic install. 

Unfortunately the way back is not as streamlined and easy as adding the community containers. Although I removed the local-ai contianer, then removed the mastercontainer, issued the run command without adding the community container, my AIO configuration page is still showing Local AI in the containers sections and is trying to start it. This is even after a reboot. Now istopped the local ai container manually.

Can someone please hint me in the right direction which configuration file to touch, so the community container is gone for good in my setup?

Thanks and kind regards
RD

**Comment by szaimen** [comment, marked as answer]
> As Pi-Hole did not start ("s6-overlay-suexec: fatal: can only run as pid 1"

This is a bug and will be fixed with https://github.com/nextcloud/all-in-one/pull/3685

---

For the rest see https://github.com/nextcloud/all-in-one/tree/main/community-containers#how-to-remove-containers-from-aios-stack

**Reply by szaimen** [reply]
The bugfix is now released with v7.6.2 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Comment by xneakersvault** [comment]
I am currently having the same issue, i have AIO running on docekr compose, i removed the variable for the community container, as stated here https://github.com/nextcloud/all-in-one/tree/main/community-containers#how-to-remove-containers-from-aios-stack, when i run my docker compose back up, the removed container appears on the AIO webinterface and when I start the containers, it pulls the image back and starts the container that i removed. This is with Face Recognition, any idea on how to fully remove ir?

**Reply by rd-911** [reply]
As I had fiddled around a bit with some community containers I had to remove afterwards, I just did the proceder few weeks ago. I had no problem with this following the instructions. Just be shure to have the space instead of the community containers in the startup command (--env AIO_COMMUNITY_CONTAINERS=" "), I remember I just deleted the containers name and did not leave a space in the quotation marks, and that I still had the containers left.

**Reply by vrjula** [reply]
Same issue here.  I'm running in a docker compose file.  Had to change to single quotes (AIO_COMMUNITY_CONTAINERS = ' ') for mastercontainer to run successfully, but the community container still shows.

**Reply by amnsharif** [reply]
for me, I had to uncomment `AIO_COMMUNITY_CONTAINERS =` and keep it blank. Did not need ' '
